library/src/main/scala/scala/scalajs/js/Any.scala

```diff
@@ -195,6 +195,14 @@ object Any extends js.LowPrioAnyImplicits {
     value.asInstanceOf[js.Any]
 
   implicit class ObjectCompanionOps(val __self: js.Object.type) extends AnyVal {
+    /** Tests whether the specified object `o` has a property `p` on itself or
+     *  in its prototype chain.
+     *
+     *  This method is the equivalent of `p in o` in JavaScript.
+     */
+    def hasProperty(o: js.Object, p: String): Boolean =
+      js.special.in(p, o)
+
```

Comment:
Why no overload for symbols?

Reply:
I was thinking I'd keep the surface API intact in this "refactoring" PR. But we could add one, yes. Or maybe better, type `p` as `String | js.Symbol`?

Reply:
Ah, ok. If you want to add it later, that's fine with me. There are probably other spots in this file where Symbol "support" should be added.

I'm not sure typing as `String | js.Symbol` is a good idea. I would expect better behavior from the Scala typer in general with overloads than implicits.